src/nginx/t/BUILD

```diff
@@ -92,6 +92,16 @@ nginx_suite(
 )
 
 nginx_suite(
+    nginx = "//src/nginx/main:nginx-esp",
```

Comment:
just add to same nginx_suite as "auth_pkey_cache.t", no need to use a separate one

Reply:
Done.